CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing
+
+## Overview
+
+* `yarn install` - install all dependencies
+* `yarn build` - run src/ through [babel] into lib/
+* `yarn test` - run all tests in repo
+* `yarn format` - run [prettier] on all files
```

Comment:
Since it's listed here, can we add `prettier` and the `format` script to this PR too?

Reply:
Will be done in #35.